lib/tools/adb-emu-commands.js

```diff
@@ -2,7 +2,14 @@ import log from '../logger.js';
 import _ from 'lodash';
 
 const PHONE_NUMBER_PATTERN = /^[\+]?[(]?[0-9]*[)]?[-\s\.]?[0-9]*[-\s\.]?[0-9]{2,}$/im;
+
 let emuMethods = {};
+emuMethods.POWER_AC_ON = 'on';
```

Comment:
Within the object? There is no way, other than to make it a getter with no setter:
```js
let emuMethods = {
  get POWER_AC_ON {
    return 'on';
  },
};
```

But in JavaScript const is not much use. It is fine just to have it there and assume no one is going to modify it
```js
let emuMethods = {
  POWER_AC_ON: 'on',
};
```